script.js

```diff
@@ -1,75 +1,96 @@
+// const rock = document.querySelector("#rock");
+// const paper = document.querySelector("#paper");
+// const scissors = document.querySelector("#scissors");
+const buttons = document.querySelectorAll("button");
+const playerScore = document.querySelector(".playerScore");
+const cpuScore = document.querySelector(".cpuScore");
+const displayMessage = document.querySelector("#displayMessage");
+
+let computerSelection;
+let humanSelection;
+let humanScore = 0;
+let computerScore = 0;
+
+playerScore.textContent = humanScore;
+cpuScore.textContent = computerScore;
+
+buttons.forEach((button) => {
+    button.addEventListener("click", (e) => {
+        computerSelection = getComputerChoice();
+        humanSelection = e.target.getAttribute("id");
+        playGame();
+    });
+});
+
+// rock.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+// paper.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+// scissors.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+
 function getComputerChoice() {
-    let cpuChoice;
+
     let choice = Math.floor(Math.random() * 3) + 1;
     
     if (choice === 1) {
-        cpuChoice = "ROCK";
+        return "rock";
     }
     else if (choice === 2) {
-        cpuChoice = "PAPER";
+        return "paper";
     }
     else {
-        cpuChoice = "SCISSORS";
+        return "scissors";
     }
-
-    return cpuChoice;
```

Comment:
I feel all possible outcomes would've been exhausted before this return cpuChoice line is reached. What do you think?

Reply:
I have removed this one also